pghoard/config.py

```diff
@@ -159,6 +160,124 @@ def read_json_config_file(filename, *, check_commands=True, add_defaults=True, c
     return set_and_check_config_defaults(config, check_commands=check_commands, check_pgdata=check_pgdata)
 
 
+def validate_config_file(config_fp):
+    config = json.load(config_fp)
+
+    NUMERIC_RE = re.compile(r"\d+")
+
+    def is_boolean(b):
+        return b in ("true", "false", "True", "False")
```

Comment:
Hmm, why isn't this just checking if it's True or False. Isn't this a python dictionary value at this point i.e. has to be an actual python boolean?

Reply:
Good point. I was unsure whether pghoard worked for both `"true"` and `"True"` but after re-reading the code, `type(b) is bool` seems like the reasonable thing to do.